css/style-contacts.css

```diff
@@ -0,0 +1,300 @@
+  border-left: 5px solid #355b82;
+  background-color: #00c4c4;
+  opacity: 1;
+  font-weight: 700;
+}
+
+.main-form__input-three:hover{
+  border-left: 5px solid #355b82;
+  background-color: #00c4c4;
+  opacity: 1;
+  font-weight: 700;
+}
+
+.main-form__input-four:hover{
+  border-left: 5px solid #355b82;
+  background-color: #00c4c4;
+  opacity: 1;
+  font-weight: 700;
+}
+
+.main-contacts__address {
+  color: #262626;
+  font-size: 20px;
+  text-align: left;
+  font-family: 'Raleway', sans-serif;
+  font-weight: 700;
+}
+
+.main-contacts__line-address { 
+  font-family: 'Raleway', sans-serif;
+  font-size: 17px;
+  font-weight: 300;
+  margin-bottom: 15px;
+}
+
+.main-contacts__email { 
+  color: #262626;
+  font-size: 20px;
+  text-align: left;
+  font-family: 'Raleway', sans-serif;
+  font-weight: 700;
+}
+
+.main-contacts__line-email { 
+  font-family: 'Raleway', sans-serif;
+  font-size: 17px;
+  font-weight: 300;
+  margin-bottom: 15px;
+}
+
+.main-contacts__phone { 
+  color: #262626;
+  font-size: 20px;
+  text-align: left;
+  font-family: 'Raleway', sans-serif;
+  font-weight: 700;
+}
+
+.main-contacts__line-phone { 
+  font-family: 'Raleway', sans-serif;
+  font-size: 17px;
+  font-weight: 300;
+}
+
+/*footer*/
+.footer { 
+  height: 90px;
+  background-color: #262626;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.footer__subtext-first { 
+  color: #b9b9b9;
+  font-family: 'Raleway', sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.footer__logo { 
+  color: #b9b9b9;
+  font-family: 'Pacifico', cursive;
+  font-size: 25px;
+  font-weight: 400;
+  text-align: right;
+}
+
+.footer__subtext-second{ 
+  color: #b9b9b9;
+  font-family: 'Raleway', sans-serif;
+  font-size: 15px;
+  font-weight: 300;
+  text-align: left;
+}
+
+.red{
+  color: #ff2b58;
+  font-family: 'Raleway', sans-serif;
+  font-weight: 700;
+}
+
+@media (max-width:1080px){
+  .contacts { 
+    background: url(../image/2ce-bakg.png) no-repeat center center;
+  }
+
+  .contacts-header__title { 
+    padding-top: 20px;
+    font-size: 50px;
+    text-align: center;
+  }
+
+  .contacts-header__text { 
+    margin: 34px auto;
+  }
+
+  .main{
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .main-form__input-four {
+    margin-bottom: 10px;
+  }
+
+  .footer { 
+    margin-top: 10px;
+    height: 45px;
+  }
+}
+
+@media (max-width: 940px) {
+  .footer { 
+    flex-direction: column;
+    height: 150px;
+  }
+}
+
+@media (max-width:684px){
+  .main-form__input-first{
+    max-width: 300px;
+  }
+  
+  .main-form__input-second{
+    max-width: 300px;
+  }
+  
+  .main-form__input-three{
+    max-width: 300px;
+  }
+  
+  .main-form__input-four {
+    max-width: 300px;
+  }
+}
+
+@media (max-width:485px){
+  .contacts { 
+    max-width: 400px;
+    margin: 0 auto;
+  }
+
+  .contacts-header__text { 
+    max-width: 300px;
+    font-size: 14px;
+    margin: 34px auto;
+  }
+}
+
+@media (max-width:320px){
```

Comment:
Are you sure those styles will get applied? Since there are not devices < 320px and our template has no design for devices less than 320px there is no need for this `@media`